configs/mot/qdtrack/README.md

```diff
@@ -50,8 +50,82 @@ The pre-trained checkpoint is given above([model](https://download.openmmlab.com
 
 b. Save the model to `ckpts/tao/**.pth`, and modify the configs for TAO accordingly(set `load_from` to your **ckpt path**).
 
+See `1.2 Example on TAO Dataset` to get more details.
+
 We observe around 0.5 track AP fluctuations in performance, and provide the best model.
 
 | Method  |   Detector   | Train Set |    Test Set    | Inf time (fps) | Track AP(50:75) | Track AP50 | Track AP75 |                         Config                         |                                                                                                                                        Download                                                                                                                                        |
 | :-----: | :----------: | :-------: | :------------: | :------------: | :-------------: | :--------: | :--------: | :----------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
 | QDTrack | Faster R-CNN | TAO train | TAO validation |       -        |      11.0       |    15.8    |    6.1     | [config](qdtrack_faster-rcnn_r101_fpn_8xb2-12e_tao.py) | [model](https://download.openmmlab.com/mmtracking/mot/qdtrack/tao_dataset/qdtrack_faster-rcnn_r101_fpn_12e_tao_20220613_211934-7cbf4062.pth) \| [log](https://download.openmmlab.com/mmtracking/mot/qdtrack/tao_dataset/qdtrack_faster-rcnn_r101_fpn_12e_tao_20220613_211934.log.json) |
+
+## Get started
+
+### 1. Training
+
+Due to the influence of parameters such as learning rate in default configuration file, we recommend using 8 GPUs for training in order to reproduce accuracy. You can use the following command to start the training.
+
+**1.1 Example on MOT Challenge Dataset**
+
+```shell
+# Training QDTrack on crowdhuman and mot17-half-train dataset with following command.
+# The number after config file represents the number of GPUs used. Here we use 8 GPUs.
+./tools/dist_train.sh \
+    configs/mot/qdtrack/qdtrack_faster-rcnn-r50_fpn_8xb2-4e_crowdhuman-mot17halftrain_test-mot17halfval.py 8
+```
+
+**1.2 Example on TAO Dataset**
+
+- a. Pre-train the QDTrack on LVISv0.5+COCO2017 training set.
```

Comment:
It's better to explain the `Results and models on LVIS dataset` in the previous section is evaluated using this model.